### PR TITLE
Fixes #23548: Integrate Sass with front-end development tools

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/23548

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/gulpfile.js [hunk 1]
```diff
@@ -9,15 +9,24 @@ const elm_p = require('gulp-elm');
...
     'css': {
-        'src': 'style/**/*',
+        'src': [
+          'style/libs/**/*',
+          'style/login.css'
+        ],
         'dest': 'webapp/style/',
     },
+    'scss': {
+        'src': 'style/rudder/**/*',
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/gulpfile.js:24]: couldn't you put the login.css file in an array in src here?

File: webapp/sources/rudder/rudder-web/src/main/gulpfile.js [hunk 2]
```diff
@@ -9,15 +9,24 @@ const elm_p = require('gulp-elm');
...
 const paths = {
     'css': {
-        'src': 'style/**/*',
+        'src': [
+          'style/libs/**/*',
+          'style/login.css'
+        ],
         'dest': 'webapp/style/',
     },
+    'scss': {
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/gulpfile.js:23]: you could make a `login_scss` with its own src and dest, and use the same `scss` function for both.

File: webapp/sources/rudder/rudder-web/src/main/style/rudder/rudder-progress-bar.scss [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+* GNU General Public License for more details.
+* 
+* You should have received a copy of the GNU General Public License
+* along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+*
+*************************************************************************************
+*/
+
+$bar-radius : 4px;
```
Comment (amousset) [inline webapp/sources/rudder/rudder-web/src/main/style/rudder/rudder-progress-bar.scss:38]: this is so much more readable!

**Comment by RaphaelGauthier** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/gulpfile.js (hunk 2 above)
Comment (RaphaelGauthier) [reply webapp/sources/rudder/rudder-web/src/main/gulpfile.js:20]: Thanks, I did that and created 2 streams in the scss function, one for the "Rudder" scss files and the other for the login scss file. Then I merged these two streams

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**